cpp/CMakeLists.txt

```
@@ -290,123 +256,178 @@ include_directories("${CMAKE_BINARY_DIR}/include"
-            src/jit/type.cpp
-            src/jit/types_h_jit.cpp
-            src/jit/parser.cpp
-            src/jit/cache.cpp
-            src/jit/launcher.cpp
-            src/transform/transform.cpp
-            src/transform/jit/code/kernel.cpp
-            src/transform/nans_to_nulls.cu
-            src/bitmask/legacy/bitmask_ops.cu
-            src/stream_compaction/apply_boolean_mask.cu
-            src/stream_compaction/drop_nulls.cu
-            src/stream_compaction/drop_duplicates.cu
-            src/datetime/datetime_ops.cu
-            src/datetime/datetime_util.cpp
-            src/hash/hashing.cu
-            src/quantiles/quantiles.cu
-            src/reductions/reductions.cu
-            src/reductions/min.cu
-            src/reductions/max.cu
-            src/reductions/sum.cu
-            src/reductions/product.cu
-            src/reductions/sum_of_squares.cu
-            src/reductions/mean.cu
-            src/reductions/var.cu
-            src/reductions/std.cu
-            src/replace/replace.cu
-            src/reductions/scan.cu
-            src/transpose/transpose.cu
-            src/merge/merge.cu
-            src/unary/math_ops.cu
-            src/unary/cast_ops.cu
-            src/io/cuio_common.cpp
-            src/io/io_functions.cpp
-            src/io/convert/csr/cudf_to_csr.cu
-            src/io/convert/dlpack/cudf_dlpack.cpp
-            src/io/avro/avro_reader_impl.cu
-            src/io/avro/avro_gpu.cu
-            src/io/avro/avro.cpp
-            src/io/csv/csv_reader_impl.cu
-            src/io/csv/csv_writer.cu
-            src/io/json/json_reader_impl.cu
-            src/io/orc/orc_reader_impl.cu
-            src/io/orc/orc.cpp
-            src/io/orc/timezone.cpp
-            src/io/orc/stripe_data.cu
-            src/io/orc/stripe_init.cu
-            src/io/parquet/page_data.cu
-            src/io/parquet/page_hdr.cu
-            src/io/parquet/parquet_reader_impl.cu
-            src/io/parquet/parquet.cpp
-            src/io/comp/cpu_unbz2.cpp
-            src/io/comp/uncomp.cpp
-            src/io/comp/brotli_dict.cpp
-            src/io/comp/debrotli.cu
-            src/io/comp/unsnap.cu
-            src/io/comp/gpuinflate.cu
-            src/io/utilities/datasource.cpp
-            src/io/utilities/parsing_utils.cu
-            src/utilities/cuda_utils.cu
-            src/utilities/column_utils.cpp
-            src/utilities/error_utils.cpp
-            src/utilities/nvtx/nvtx_utils.cpp
-            src/copying/copy.cpp
-            src/copying/gather.cu
-            src/copying/scatter.cu
-            src/copying/slice.cu
-            src/copying/split.cu
-            src/bitmask/legacy/legacy_bitmask.cpp
-            src/gis/point_in_polygon.cu
-            src/copying/copy_range.cu
-            src/filling/fill.cu
-            src/search/search.cu)
-
-#Override RPATH for cudf
-SET_TARGET_PROPERTIES(cudf PROPERTIES BUILD_RPATH "\$ORIGIN")
+
+add_library(libNVStrings
```

According to the docs for `add_library` (https://cmake.org/cmake/help/v3.12/command/add_library.html): 

> The actual file name of the library built is constructed based on conventions of the native platform (such as lib<name>.a or <name>.lib)."

So shouldn't this be:

```suggestion
add_library(NVStrings
```

Otherwise it will create a library called `liblibNVStrings.a`? Notice that we have `add_library(cudf` below, not `add_library(libcudf`.